REQUIRE

```diff
@@ -0,0 +1,2 @@
+julia 0.3
+Compat
```

Comment:
that's not easy but the most important thing
in the middle of tracking & following and sleuthing the history of the decisions.
clean test makes me happy though.